python/pyspark/sql/column.py

```diff
@@ -527,7 +584,7 @@ def _test():
         .appName("sql.column tests")\
         .getOrCreate()
     sc = spark.sparkContext
-    globs['sc'] = sc
+    globs['spark'] = spark
```

Comment:
I removed `sc` and replaced it to `spark` as I think we promote this way up to my knowledge.